scripts/db/create_test_db.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-./scripts/db/import_db.sh -H localhost -P 3306 -u test_bot -p pass -d test_results_db -l ./test_results_db.sql
+./scripts/db/import_db.sh -H localhost -P 3306 -u test_bot -p pass -d test_results_db -l ./migration-0.sql
```

Comment:
This script is responsible for creation of the whole database. So it must execute SQL code that crates the latest version of the database, not the first one.